source/nvwave.py

```diff
@@ -640,3 +663,8 @@ def _cleanup():
 	global fileWavePlayer, fileWavePlayerThread
 	fileWavePlayer = None
 	fileWavePlayerThread = None
+
+
+@property

```

Comment:
I believe this is a mistake. Modules cannot have properties.

Reply:
Yes, I think a result of moving code out of the class. I'll fix this.